Fix duplicate success notifications in contact form

rajgokul45300 opened the pull request:
- Remove duplicate .success-alert div from HTML
- Fix CSS display property conflict for .alert class
- Update JavaScript to use display: flex instead of display: block
- Ensure only one success notification appears when form is submitted

Fixes issue #27: Message sent notification appears 2 times

Before:
<img width="1719" height="1039" alt="Screenshot 2025-10-01 at 4 04 06 PM" src="https://github.com/user-attachments/assets/3edfe822-1dcd-4510-ac61-9f3847796508" />

After:
<img width="1564" height="1026" alt="Screenshot 2025-10-01 at 4 04 25 PM" src="https://github.com/user-attachments/assets/fc707669-bc22-43a7-9fc8-97cef996698d" />

Commit:
Fix duplicate success notifications in contact form

- Remove duplicate .success-alert div from HTML
- Fix CSS display property conflict for .alert class
- Update JavaScript to use display: flex instead of display: block
- Ensure only one success notification appears when form is submitted

Fixes issue #27: Message sent notification appears 2 times

yesiamrajeev reviewed (changes requested):
Its looking good now
but can you align some boxes there?

Commit:
Improve contact form alignment and styling

- Fix grid layout alignment issues
- Revert forced stretching that caused misalignment
- Improve spacing and proportions for better visual balance
- Adjust textarea padding for better form appearance
- Ensure contact info cards and form have natural, balanced sizing

Continues fix for issue #27: Message sent notification appears 2 times

rajgokul45300 reviewed (commented):
Improve contact form alignment and styling
- Fix grid layout alignment issues
- Revert forced stretching that caused misalignment
- Improve spacing and proportions for better visual balance
- Adjust textarea padding for better form appearance
- Ensure contact info cards and form have natural, balanced sizing

Continues fix for issue https://github.com/yesiamrajeev/Hacktoberfest2025-Portfolio/issues/27: Message sent notification appears 2 times
<img width="1545" height="1015" alt="Screenshot 2025-10-01 at 4 45 58 PM" src="https://github.com/user-attachments/assets/593151c0-a65f-4b10-b0d4-c9dd5b7ac49d" />

yesiamrajeev reviewed (approved):
Approved

yesiamrajeev commented:
Add yourself in contributors.json
